schema/Address.ts

```diff
@@ -8,15 +9,16 @@ export const TicketedEventDetailsInput = inputObjectType({
     t.nonNull.string("name", {
       description: "The name of the event. IE My Festival",
     });
-    t.string("venue", {
-      description: "The venue for the event. IE On an Island",
-    });
-    t.string("description", {
-      description: "The description of the event. IE Fun on an Island",
-    });
-    t.date("dateTime", {
-      description: "The date and time of the event in ISO date format.",
-    });
+    // When we initially create an event its just a collection
```

Comment:
maybe leave these as optional?

Reply:
Its an issue of where do we store the data, something im going to bring up with Jordy, you really should be able to store more info on the collection then just its name.